let User,CustomErrorHandler;_7b2‍.x([["default",()=>_7b2‍.o]]);_7b2‍.w("./../../models",[["User",["User"],function(v){User=v}]]);_7b2‍.w("./../../services/CustomeErrorHandler",[["default",["CustomErrorHandler"],function(v){CustomErrorHandler=v}]]);

const userController = {
  async me(req, res, next) {
    try {
      const user = await User.findOne({ _id: req.user._id }).select(
        "-password -updatedAt -__v"
      );
      if (!user) {
        return next(CustomErrorHandler.notFound());
      }

      res.json(user);
    } catch (err) {
      return next(err);
    }
  },
};

_7b2‍.d(userController);
